src/app/gm-tools/feature/create-bracket-page.component.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  WritableSignal,
+  computed,
+  inject,
+} from '@angular/core';
+import { PageHeaderComponent } from '../../shared/ui/page-header.component';
+import { HeaderLinkComponent } from '../../shared/ui/header-link.component';
+import { ButtonModule } from 'primeng/button';
+import { FormsModule } from '@angular/forms';
+import { InputNumberModule } from 'primeng/inputnumber';
+import { TournamentBracketComponent } from '../../shared/ui/tournament-bracket.component';
+import { EventService } from '../../shared/data-access/event.service';
+import { DropdownModule } from 'primeng/dropdown';
+import { EventModel } from '../../shared/util/supabase-types';
+
+@Component({
+  selector: 'joshies-create-bracket-page',
+  standalone: true,
+  template: `
+    <joshies-page-header headerText="Bracket Test Page" alwaysSmall>
+      <joshies-header-link
+        text="GM Tools"
+        routerLink=".."
+        chevronDirection="left"
+      />
+    </joshies-page-header>
+    <div class="flex flex-column mt-5 font-semibold">
+      @if (events()) {
+        <p-dropdown
+          [options]="events()!"
+          [(ngModel)]="selectedEvent"
+          optionLabel="id"
+          placeholder="Select an Event"
+          styleClass=" w-full"
+        >
+          <ng-template let-ev pTemplate="selectedItem">
+            <div class="flex align-items-center gap-2">
+              <div>{{ ev.name }}</div>
+            </div>
+          </ng-template>
+          <ng-template let-ev pTemplate="item">
+            <div class="flex align-items-center gap-2">
+              <div>{{ ev.name }}</div>
+            </div>
+          </ng-template>
+        </p-dropdown>
+      }
+    </div>
+    <joshies-tournament-bracket [eventId]="selectedEvent.id!" />
```

Comment:
Refactored to use an `@if` block.  `selectedEvent` is now also `selectedEventId` of type `number | null` because the bracket component only needs an `eventId` as input.